README.md

```diff
@@ -90,12 +90,74 @@ When you have addressed all technical review and peer review comments, notify th
 
 The developers check that the **Tech review passed** label has been added to the pull request and peer pull request approval provided, then accept it.
 
-## Installing Pantheon
+## Deploying Pantheon
 
-### Prerequisites
+Install the Pantheon codebase, perform some basic initial configuration, and deploy Pantheon with a containerized MongoDB database.
+
+**Prerequisites**
+* Tested on Red Hat Enterprise Linux (RHEL) 8.3 and Fedora 33.
+* openJDK
```

Comment:
Which OpenJDK?

Reply:
And incidentally whichever version is used, make sure to install *all* the packages, not just the java-<ver>-openjdk, because it does not pull in all of the dependencies required.

Reply:
for Fedora, java-1.8.0-openjdk

Reply:
for docker image, 

FROM registry.access.redhat.com/ubi8/ubi
RUN yum -y install java-1.8.0-openjdk

Reply:
please note, we have a story in the backlog for us to upgrade to java 11 in the near future.

Reply:
That's what I tried first, but had issues with unclear module definition in the current build because of (I think?) changes in JDK9.